main.go

```diff
@@ -39,12 +35,28 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
 
 	apiv1 "github.com/redhat-developer/observability-operator/v3/api/v1"
 	"github.com/redhat-developer/observability-operator/v3/controllers"
 	// +kubebuilder:scaffold:imports
 )
 
+type OperandInitializer struct {
```

Comment:
you can move those definitions to a separate file under a new directory `runners`.

Reply:
Something like `runners/runners.go` ?

Reply:
`runners/generic_runner.go`

Reply:
Will do. Thanks